generators/app/prompts.js

```diff
@@ -216,3 +217,50 @@ function askModulesToBeInstalled(done, generator) {
         }
     );
 }
+
+function askForJavaVersion() {
+    if (this.existingProject) {
+        return;
+    }
+
+    const choices = [{ name: 'Java 11', value: '11' }, { name: 'Java 8', value: '1.8' }];
+    const defaultChoice = [{ name: 'Java 11', value: '11' }];
+    const PROMPT = {
+        type: 'list',
+        name: 'javaVersion',
+        message: 'Which Java version would you like to use?',
+        choices,
+        default: defaultChoice
+    };
+
+    const done = this.async();
+
+    this.prompt(PROMPT).then(prompt => {
+        this.javaVersion = prompt.javaVersion;
+        if (prompt.javaVersion === '11') {
+            askJacksonBlackBirdToBeInstalled(done, this);
+        } else {
+            done();
+        }
+    });
+}
+
+function askJacksonBlackBirdToBeInstalled(done, generator) {
+    const PROMPT = {
+        type: 'input',
```

Comment:
This should be a type "confirm" using a boolean

Reply:
@cbornet : Thanks. Done 👍